src/server.c

```diff
@@ -150,62 +116,16 @@ void
 sig_handler(int sig) {
     if (force_exit)
         exit(EXIT_FAILURE);
-
-    char sig_name[20];
-    get_sig_name(sig, sig_name);
-    lwsl_notice("received signal: %s (%d), exiting...\n", sig_name, sig);
+    lwsl_notice("received signal: %s (%d), exiting...\n", strsignal(sig), sig);
```

Comment:
this is a mistake.
The behavior is not change, 
refusing this diff is a big problem to me 
i do not see how the previous code would be any better.

The log is still here and the SIGNAL NUMBER wont be translated by a dubious lib anyway